plugins/module_utils/utils.py

```diff
@@ -202,5 +195,5 @@ def make_op(path, old, new, strategy):
             new = dict(old, **new)
     elif isinstance(old, list):
         if strategy == "merge":
-            new = list_merge(old, new)
+            new = list(set(old + new))
```

Comment:
We can't use `set()` here because the list items might be unhashable, like dicts.